core/modules/css/CssAccess.cc

```diff
@@ -503,7 +503,8 @@ CssAccess::getPartTableParams(std::string const& dbName,
 
     std::vector<std::string> subKeys{"partitioning", "partitioning/subChunks",
         "partitioning/dirDb", "partitioning/dirTable", "partitioning/dirColName", "partitioning/latColName",
-        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName"};
+        "partitioning/lonColName", "partitioning/overlap", "partitioning/secIndexColName",
+        "partitioning/lockInMem", "partitioning/scanSpeed"};
```

Comment:
No, I suggest we stick with current PartTableParams for now, but be prepared to separate them in the future. CSS will evolve and I don't assume that what we do today will be the final solution, it depends a lot on how these parameters are used or updated. I guess what I'm saying let's implement simplest thing today but be prepared to change it when we understand it better. If you want to make separate structure for it it's OK with me but then we need to think about CssAccess interface as well, because that structure is used differently from regular partitioning parameters.